lib/active_merchant/billing/gateways/priority.rb

```diff
@@ -72,6 +72,17 @@ def authorize(amount, credit_card, options = {})
         commit('purchase', params: params)
       end
 
+      def credit(amount, credit_card, options = {})
```

Comment:
Thank you for making a new branch/PR. I should have asked for this sooner, but can you link to the gateway docs where it is referencing `credit`? I tried looking [here](https://developer.mxmerchant.com/docs) but could not find documentation for this transaction type.  I ask because of the question about a negative amount for credit. I've done some investigating and can't find that to be true for other gateways. Thank you!